packages/gluestick/src/commands/start-client.js

```diff
@@ -1,34 +1,58 @@
+/* @flow */
+import type { Context, WebpackConfigEntry, Compiler } from '../types.js';
+
 const path = require('path');
 const webpack = require('webpack');
 const express = require('express');
 const proxy = require('http-proxy-middleware');
 
-module.exports = ({ config, logger }) => {
-  const { GSConfig } = config;
-  const configuration = config.webpackConfig.client;
-  const developmentServerOptions = {
-    quiet: true, // don’t output anything to the console
-    noInfo: true, // suppress boring information
-    hot: true, // adds the HotModuleReplacementPlugin and switch the server to hot mode.
-    // Note: make sure you don’t add HotModuleReplacementPlugin twice
-    inline: true, // also adds the webpack/hot/dev-server entry
-
-    // You can use it in two modes:
-    // watch mode (default): The compiler recompiles on file change.
-    // lazy mode: The compiler compiles on every request to the entry point.
-    lazy: false,
+type DevelopmentServerOptions = {
+  quiet: boolean, // don’t output anything to the console
+  noInfo: boolean, // suppress boring information
+  hot: boolean, // adds the HotModuleReplacementPlugin and switch the server to hot mode.
+  // Note: make sure you don’t add HotModuleReplacementPlugin twice
+  inline: boolean, // also adds the webpack/hot/dev-server entry
+  // You can use it in two modes:
+  // watch mode (default): The compiler recompiles on file change.
+  // lazy mode: The compiler compiles on every request to the entry point.
+  lazy: boolean,
+  // network path for static files: fetch all statics from webpack development server
+  publicPath: string,
+  headers: Object,
+  stats: Object,
+};
+
+module.exports = ({ config: { GSConfig, webpackConfig }, logger }: Context): void => {
+  if (!webpackConfig) {
+    logger.error('Webpack config not specified');
+    return;
```

Comment:
Yea i was actually reading https://www.nczonline.net/blog/2014/02/04/maintainable-node-js-javascript-avoid-process-exit/ and probably we should thrown errors instead of following the `process.exit(1)` approach that is used in some parts of the code (+ this `process.exit` gives problems in test if we do not mock it).